src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -82,21 +67,16 @@
 import com.owncloud.android.operations.UpdateShareViaLinkOperation;
 import com.owncloud.android.services.OperationsService;
 import com.owncloud.android.services.OperationsService.OperationsServiceBinder;
-import com.owncloud.android.ui.NavigationDrawerItem;
-import com.owncloud.android.ui.adapter.NavigationDrawerListAdapter;
 import com.owncloud.android.ui.dialog.ConfirmationDialogFragment;
 import com.owncloud.android.ui.dialog.LoadingDialog;
 import com.owncloud.android.ui.dialog.SslUntrustedCertDialog;
 import com.owncloud.android.utils.ErrorMessageAdapter;
 
-import java.util.ArrayList;
-
 
 /**
- * Activity with common behaviour for activities handling {@link OCFile}s in ownCloud
- * {@link Account}s .
+ * Activity with common behaviour for activities handling {@link OCFile}s in ownCloud {@link Account}s .
  */
-public class FileActivity extends AppCompatActivity
+public class FileActivity extends DrawerActivity
```

Comment:
Yes that was basically the idea and yes, composition would be preferable and yes that should be done in another PR